oat-sa/tao-core
# fix: TR-3292. Restore focusing functionality

Related to: https://oat-sa.atlassian.net/browse/TR-3292

This style change fix problem with breaking focusing flow for pages with checkboxes.
Root cause of this - `visibility:hidden` style do not allow focus element. This style was added in terms of fixing this: https://oat-sa.atlassian.net/browse/TR-916. After testing with author we decided the changing `opacity` instead of `visibility` is a good compromise for both tickets.

TODO:
- [x] update tao-core-ui-fe dependency from https://github.com/oat-sa/tao-core-ui-fe/pull/407 after releasing 

How to test:

0. You can check full flow by running `npm ci` and `npx grunt taosass` before running the test
1. run test with choice interaction
2. move focus using keyboard
3. be sure focus jumps and console without errors
